crates/topos/src/components/regtest/commands/spam.rs

```diff
@@ -67,6 +67,21 @@ pub struct Spam {
     /// If not provided open telemetry will not be used
     #[arg(long, env = "TOPOS_OTLP_SERVICE_NAME")]
     pub otlp_service_name: Option<String>,
+    /// Flag to indicate usage of Kubernetes.
+    #[arg(
+        long,
+        env = "TOPOS_NETWORK_SPAMMER_BENCHMARK",
+        requires = "hosts",
+        requires = "number"
+    )]
+    pub benchmark: bool,
+    /// Template for generating target node entrypoints.
```

Comment:
maybe the variable name should also be `entrypoints` instead of `hosts`

Reply:
How about `target_hosts`?

Reply:
I'll go with `target_hosts`